### ISSUE TITLE
Strange test failure in tests/acceptance/generate-test.js

### ISSUE THREAD

**Issue by joliss** [ember-cli/ember-cli#4751]
When I run `npm run test-all`, I get the following test failure:

```
  1) Acceptance: ember generate passes custom cli arguments to blueprint options:

      expected app/foo.js to exist
      + expected - actual

      -false
      +true

      at assertFile (tests/helpers/assert-file.js:43:35)
      at tests/acceptance/generate-test.js:1151:9
```

But I don't get this failure when I run `mocha tests/acceptance/generate-test.js`.

I'm not sure why this is happening, and it's pretty hard to debug because it takes a very lengthy test run to reproduce. Ideas welcome!

Edit: Investigating some more...

**Comment by joliss** [comment]
It seems that this is fixed by my very own #4747, which I forgot to apply. Lol.

I'll reopen if I see this again.

**Comment by stefanpenner** [comment]
> I'll reopen if I see this again.

I suspect you forgot to close it, will gladly reopen if it turns out to be another issue.
